openvpn/windscribe/Adelaide-Lofty-tcp.ovpn

```diff
@@ -5,7 +5,7 @@ remote adl-354.whiskergalaxy.com 1194
 verify-x509-name adl-354.windscribe.com name
 
 nobind
-auth-user-pass /config/openvpn-credentials.txt
```

Comment:
@pkishino should these lines have been changed?

Reply:
Yeah, that’s fine.. there mod script that runs per default fixes this

Reply:
Yes I think with the last dump I ran the adjustConfigs script, that seems to be not needed anymore :)